### PR TITLE
add Determine the splicing result according to the given registry addresses of different types

### PR DESCRIPTION
**Pull request by pipipipipi43**
…resses of different types

#### What type of this PR

Add one of the following kinds:

/kind bug

#### What this PR does / why we need it:

add Determine the splicing result according to the given registry addresses of different types.

registry/org-project/app:tag =>  registry/org-project-app:tag

#### Which issue(s) this PR fixes:

- [Erda Cloud Issue Link](https://terminus-org.app.terminus.io/erda/dop/projects/387/issues/bug?id=220319&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6InRhYmxlIiwiY2hpbGRyZW5WYWx1ZSI6eyJrYW5iYW4iOiJkZWFkbGluZSJ9fQ%3D%3D&iterationID=0&type=BUG)

### CODE REVIEW

File: modules/pipeline/services/pipelinesvc/secret_test.go [hunk 1]
```diff
@@ -71,3 +72,59 @@ func TestAddRegistryLabel(t *testing.T) {
 		})
 	}
 }
+
+func TestReplaceProjectApplication(t *testing.T) {
+	type args struct {
```
Comment (Effet) [inline modules/pipeline/services/pipelinesvc/secret_test.go:77]: what if case is "addon-registry.default.svc.cluster.local:5000/", please show me then how it works.

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/pipeline/services/pipelinesvc/secret_test.go (hunk 1 above)
Comment (pipipipipi43) [reply modules/pipeline/services/pipelinesvc/secret_test.go:77]: addon-registry.default.svc.cluster.local:5000//project-application : tag

**Review by Effet — APPROVED** [review]

**Pull request merged**